### DISCUSSION TITLE
Is it allowed to contribute on new theme for FreshRSS?

### DISCUSSION THREAD

**Discussion by shinemoon** [FreshRSS/FreshRSS#5849] [Q&A] [answered]
Hi,

I am just a bit curious after checking contributor guideline and seems no info there about submit new theme for FreshRSS (or I missed some info..), so is there some guideline for that , or even allowed? :)

Thanks!

**Comment by Alkarex** [comment, marked as answer]
Sure, go ahead!
Start by checking the practices used by recently updated themes such as *Origine* and *Nord*, in particular regarding the use of CSS variables. Maybe @math-GH can develop some more

**Reply by shinemoon** [reply]
Thanks, let me try :)

**Reply by math-GH** [reply]
First lines of a little guideline for contributing a new system theme is written now. Check it out: https://freshrss.github.io/FreshRSS/en/contributing.html#contribute-to-the-system-themes

**Reply by shinemoon** [reply]
It sounds at least I contributed a little indirectly on document enhancement? ;)

**Reply by math-GH** [reply]
yes :D
